ag_data/formulas/pipeline.py

```diff
@@ -30,21 +31,36 @@ def save_measurement(self, sensor, timestamp, measurement) -> AGMeasurement:
             if active:
                 event = active.agevent
```

Comment:
In case we just want to record error only if there is no active event, how about just adding else statement here?
```
if active:
    event = active.agevent
else:
    # Currently no active event, error should be recorded
    data = {
        sensor_id=sensor

...

    record.save_error(
        raw_data=str(data),
        error_code=record.ERROR_CODE["NO_ACT_EVENT"],
        error_description="Currently no active event",
    )
    return
```

Apart from that, there is still possibility of error when we run `formulas.formula_map.get(...)`, `preprocess(...)` and so on. I'm not sure whether we should log errors for those stuffs. But if the scope of this PR is just to record errors for no active event, I think this else statement will be enough.